hbw/app/javascript/packs/hbw/init/date_localizer.js

```diff
@@ -1,9 +1,14 @@
 import { parseISO, format } from 'date-fns';
 import en from 'date-fns/locale/en-US';
 import ru from 'date-fns/locale/ru';
+import { registerLocale } from 'react-datepicker';
 
 const locales = { en, ru };
 
+Object.keys(locales).forEach((locale) => {
+  registerLocale(locale.toString(), locales[locale]);
+});
```

Comment:
ideally, it should be at a later stage, like on a 'dom-loaded' event, smth like that